gap/small.gi

```diff
@@ -37,6 +37,20 @@ InstallGlobalFunction( SMALL_AVAILABLE, function( size )
     return fail;
 end );
 
+InstallGlobalFunction( SmallGroupsAvailable,
+function(order)
+    return SMALL_AVAILABLE(order) <> fail;
+end);
+
+InstallGlobalFunction( NumberSmallGroupsAvailable,
+function(order)
+    if order = 1024 then
+        return true;
+    else
+        return SmallGroupsAvailable(order);
+    fi;
+end);
```

Comment:
I plan an extension of the SGL which adds many group orders for which no list of groups is available. For this, this function will need to be changed (e.g. by turning it into an operation). But that can be done later, of course.

Reply:
While I stand by this, of course also `NumberSmallGroups` hardcodes the 1024, so I guess more changes will be needed. No reason to block this PR over that.